modules/installing-aws-load-balancer-operator-cli.adoc

```diff
@@ -0,0 +1,133 @@
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator-cli_{context}"]
+= Installing the AWS Load Balancer Operator with the command line
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Your cluster is configured with AWS as the platform type and cloud provider.
+
+* You are logged into the command line client `oc`.
+
+.Procedure
+
+. Create the `aws-load-balancer-operator` namespace:
++
+[source,terminal]
+----
+oc apply -f namespace.yaml
+cat namespace.yaml
+----
++
+Edit the namespace to include:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aws-load-balancer-operator
+----
+
+. Create the `credentialsrequest` file:
++
+[source, terminal]
+----
+oc apply -f credentialsrequest.yaml
+cat credentialsrequest.yaml
+----
++
+Edit the file to include:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: aws-load-balancer-operator
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+      - action:
+          - ec2:DescribeSubnets
+        effect: Allow
+        resource: "*"
+      - action:
+          - ec2:CreateTags
+          - ec2:DeleteTags
+        effect: Allow
+        resource: arn:aws:ec2:*:*:subnet/*
+      - action:
+          - ec2:DescribeVpcs
+        effect: Allow
+        resource: "*"
+  secretRef:
+    name: aws-load-balancer-operator
+    namespace: aws-load-balancer-operator
+  serviceAccountNames:
+    - aws-load-balancer-operator-controller-manager
+----
+
+. Create the `operatorgroup` file:
++
+[source,terminal]
+----
+oc apply -f operatorgroup.yaml
+cat operatorgroup.yaml
+----
++
+Edit the file to include:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: aws-lb-operatorgroup
+  namespace: aws-load-balancer-operator
+spec:
+  upgradeStrategy: Default
+----
+
+. Create the `subscription` file:
++
+[source,terminal]
+----
+oc apply -f subscription.yaml
+cat subscription.yaml
+----
++
+Edit the file to include:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: qe-app-registry
+  sourceNamespace: openshift-marketplace
+----
+
+.Verification
++
+[source,terminal]
+----
+oc -n aws-load-balancer-operator get sub aws-load-balancer-operator --template='{{.status.installplan.name}}{{"\n"}}'
+oc -n aws-load-balancer-operator get ip <install_plan_name> --template='{{.status.phase}}{{"\n"}}'
+----
++
+The output must be `Complete`. 
```

Comment:
Not critical, but you could consider showing an `.Example output` code block of what the expected output for these commands are. It's not clear, since there are two commands, and I expect one of them is expected to show "Complete". But not sure if both of them are, or just one of them.